### PR TITLE
Sonatype sbt updates

### PR DESCRIPTION
**Pull request by tomerk**
@shivaram needed for maven central releases

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -56,3 +60,31 @@ mergeStrategy in assembly <<= (mergeStrategy in assembly) { (old) =>
...
+
+// To publish on maven-central, all required artifacts must also be hosted on maven central.
+pomIncludeRepository := { _ => false }
+
+pomExtra := (
+    <scm>
+      <url>git@github.com:amplab/ml-matrix.git</url>
+      <connection>scm:git:git@github.com:amplab/ml-matrix.git</connection>
+    </scm>
+        <developers>
```
Comment (shivaram) [inline build.sbt]: Style nit: could you align the `developers` with `scm` tag

File: build.sbt [hunk 2]
```diff
@@ -56,3 +60,31 @@ mergeStrategy in assembly <<= (mergeStrategy in assembly) { (old) =>
...
+          <developer>
+            <id>shivaram</id>
+            <name>Shivaram Venkataraman</name>
+            <url>http://shivaram.org</url>
+          </developer>
+        </developers>)
```
Comment (shivaram) [inline build.sbt]: Similarly could you move the closing `)` to the next line here ?

**Comment by tomerk** [comment]
@shivaram how's it look now?

**Comment by shivaram** [comment]
LGTM. Thanks @tomerk Merging this

**Pull request merged**